src/native/minipal/cpufeatures.c

```diff
@@ -563,7 +563,43 @@ int minipal_getcpufeatures(void)
         }
     }
 
-#endif // HAVE_HWPROBE_H
+#else
+
+    // https://docs.kernel.org/arch/riscv/uabi.html
+    // Example:
+    // isa             : rv64imadc_zifoo_zigoo_zafoo_sbar_scar_zxmbaz_xqux_xrux
+
+    FILE*  cpuInfo = fopen("/proc/cpuinfo", "r");
+    size_t n       = 0;
+    char*  buf     = NULL;
+
+    assert(cpuInfo != NULL);
```

Comment:
Linux `procfs` is considered optional for [number of reasons](https://github.com/dotnet/runtime/issues/62513).

```suggestion
    if (cpuInfo == NULL) return result;
```
(or wrap the handling under `!= NULL`)